web/modules/weather_data/src/Service/WeatherDataService.php

```diff
@@ -892,4 +902,12 @@ public function getDailyForecastFromGrid(
             "extended" => array_values($extendedPeriodsFormatted),
         ];
     }
+
+    /**
+     * Convert C to F
+     */
+    private function celciusToFahrenheit(float $degreesC)
+    {
+        return $degreesC * 1.8 + 32;
+    }
```

Comment:
A potential future refactor here as well, in that we should be able to replace this with the UnitConversionTrait that handles reading the input units from the data.

So instead of:
```php
"dewpoint" => $this->celsiusToFahreneith($period->dewpoint->value)
```

you'd have:
```php
"dewpoint" => $this->getTemperatureScalar($period->dewpoint)
```

And that should work even if the API suddenly started serving Fahrenheit.